starter-code/css/main.css

```diff
@@ -5,6 +5,83 @@ light blue: #1c5380
 */
 
 body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
 }
+
+.upper-block {
+  background-color: #06365f;
+  height: 400px;
+}
+
+.bottom-block {
+  color: #1c5380;
+  }
+
+.frame {
+  background: url(../images/frame.png);
+  background-repeat: no-repeat;
+  position: relative;
+  left: 590px;
+  top: -80px;
+  background-size: 100%;
+  padding-left: 35px;
+  padding-top: 25px;
+  padding-right: 10px;
+  width:450px;
+  }
+
+.phone-picture {
+  position: relative;
+  top: 170px;
+  left: 200px;
+}
+
+.logo {
+  position: relative;
+  left: 550px;
+  top: -350px;
+}
+
+.home-button {
```

Comment:
Adding a margin-left would help separate the button from the Instagram logo: as it is right now the elements look very cramped together.